egui_glium/src/lib.rs

```diff
@@ -185,6 +190,38 @@ pub fn input_to_egui(
                 input_state.raw.scroll_delta += delta;
             }
         }
+        WindowEvent::TouchpadPressure {
+            // device_id,
+            // pressure,
+            // stage,
+            ..
+        } => {
+            // TODO
+        }
+        WindowEvent::Touch(touch) => {
+            let mut hasher = std::collections::hash_map::DefaultHasher::new();
+            touch.device_id.hash(&mut hasher);
+            input_state.raw.events.push(Event::Touch {
+                device_id: hasher.finish(),
+                id: touch.id,
+                phase: match touch.phase {
+                    glutin::event::TouchPhase::Started => egui::TouchPhase::Start,
+                    glutin::event::TouchPhase::Moved => egui::TouchPhase::Move,
+                    glutin::event::TouchPhase::Ended => egui::TouchPhase::End,
+                    glutin::event::TouchPhase::Cancelled => egui::TouchPhase::Cancel,
+                },
+                pos: Pos2::new(touch.location.x as f32, touch.location.y as f32),
```

Comment:
`touch.location` is a `PhysicalPosition` so it should be divided by `pixels_per_point`

https://docs.rs/winit/0.24.0/winit/event/struct.Touch.html

Reply:
fixed by https://github.com/emilk/egui/pull/306/commits/e4c9a65d57560385fa35701fe5119a6de210e840